docs/cloud/cloud-org/cloud-org-upgrade-to-paid.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Upgrade your organization to paid
+layout: default
+parent: Manage organization
+grand_parent: Cloud
+nav_order: 4
+---
+
+# How do I upgrade my organization to paid?
+
+{: .no_toc }
+
+{% include page-toc.md %}
+
+## Before you begin
+
+{% include /cloud/cloud-before-begin.md %}
+
+## Upgrade your organization to Paid
+
+- Click **Configuration** > **Upgrade to Paid**.
+- Insert your credit card information or your bank account details.
```

Comment:
You don't insert credit card information or bank account details. "insert" is a physical action. You can insert your card to an ATM for instance.

You enter credit card information or bank account details